apps/artemis-web/libs/passport-strategies/remember-me-strategy.ts

```diff
@@ -3,14 +3,11 @@ import { getRandomString } from '../../utils/token';
 import { Db, MongoClient } from 'mongodb';
 
 const MONGODB_URI = `mongodb://${process.env.MONGODB_USER}:${process.env.MONGODB_PASS}@${process.env.MONGODB_HOST}:${process.env.MONGODB_PORT}`;
-const client = new MongoClient(MONGODB_URI, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-});
+const client = new MongoClient(MONGODB_URI, {});
 
 let dbInstance: Db = null;
 
-if (!client.isConnected()) client.connect();
+client.connect();
```

Comment:
does this fail if already connected? Or does it gracefully continue?

Reply:
No I think it continues